pkg/util/ec2/ec2_no_tags.go

```diff
@@ -11,3 +11,7 @@ package ec2
 func GetTags() ([]string, error) {
 	return []string{}, nil
```

Comment:
Why not do `return fetchTagsFromCache()` here since they both return the same thing?